generative_ai/prompts/prompt_load_or_retrieve.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+
+
+def get_prompt() -> str:
```

Comment:
Return type hint should be a `Prompt` object (yes, when you print the response it is printed as as string, but that does not mean that the return type is a string. The Prompt object displays as a string when printed)

Reply:
Marking unresolved, the return type hint is still shown as str.